.github/workflows/releaser.yml

```diff
@@ -0,0 +1,44 @@
+name: Library Releaser
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  releaser:
+    name: build_releaser
+    runs-on: ubuntu-latest
+    env:
+      BUILD_TARGETS: aarch64,armv7,x86_64,i686
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set default toolchain
+        run: rustup default stable
+      - name: Set profile
+        run: rustup set profile minimal
+      - name: Add targets
+        run: rustup target add x86_64-linux-android i686-linux-android aarch64-linux-android armv7-linux-androideabi
+      - name: Build
+        run: ./gradlew build
+      - name: release
+        uses: actions/create-release@v1
+        id: create_release
+        with:
+          draft: false
+          prerelease: false
+          release_name: bdk-jni-${{ github.ref }}
+          tag_name: ${{ github.ref }}
+          body_path: CHANGELOG.md
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
```

Comment:
In the `create-release@v1` example workflow they suggest using the `GITHUB_TOKEN` which should be provided by github actions. Can't we also use that instead of creating a special `RELEASE_TOKEN` ?

Reply:
Oh yes! So, I put this out of habit cause the `GITHUB_TOKEN` does not have access to create a release, so i wasn't sure if the CI will be able to create a release under a org, however it might work under a personal repo. I will check if that's still the case if we move to https://github.com/softprops/action-gh-release